images/README.md

```diff
@@ -1 +1,11 @@
-This repo contains all windows equivalent of docker images used to run k8s e2e tests.
\ No newline at end of file
+This repo contains all windows equivalent of docker images used to run k8s e2e tests.
```

Comment:
I don't think these docker files are up to date any more.  Should probably point folks at the ones in k/k/test/images: https://github.com/kubernetes/kubernetes/tree/master/test/images